generators/client/templates/angular/src/main/webapp/app/admin/admin.module.ts.ejs

```diff
@@ -27,7 +27,7 @@ import { <%=angularXAppName%>SharedModule } from 'app/shared';
 
 import {
     adminState,
-    <%_ if (databaseType !== 'no' && databaseType !== 'cassandra') { _%>
+    <%_ if ((databaseType !== 'no' ? true : authenticationType === 'uaa') && databaseType !== 'cassandra') { _%>
```

Comment:
I think that's just `(databaseType !== 'no'  || authenticationType === 'uaa')`

Reply:
Indeed!